### PR TITLE
ZEPPELIN-1853. Add README for livy interpreter

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?
Add `README` file for developer, besides update the `livy.md` for the recent changes on livy interpreter. 

### What type of PR is it?
[Documentation]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-1853

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by zjffdu** [comment]
@AhyoungRyu Please help review it. Thanks

**Comment by prabhjyotsingh** [comment]
+1 for having more doc.

File: docs/interpreter/livy.md [hunk 1]
```diff
@@ -51,11 +51,6 @@ Example: `spark.master` to `livy.spark.master`
     <th>Description</th>
   </tr>
   <tr>
-      <td>livy.spark.master</td>
-      <td>local[*]</td>
-      <td>Spark master uri. ex) spark://masterhost:7077</td>
-    </tr>
-  <tr>
     <td>zeppelin.livy.url</td>
```
Comment (AhyoungRyu) [inline docs/interpreter/livy.md]: There is still `livy.spark.master` mention under this section: http://zeppelin.apache.org/docs/0.7.0-SNAPSHOT/interpreter/livy.html#configuration Seems this needs to be updated accordingly.

File: docs/interpreter/livy.md [hunk 2]
```diff
@@ -205,3 +200,7 @@ Edit `conf/spark-blacklist.conf` file in livy server and comment out `#spark.mas
 
 If you choose to work on livy in `apps/spark/java` directory in [https://github.com/cloudera/hue](https://github.com/cloudera/hue),
 copy `spark-user-configurable-options.template` to `spark-user-configurable-options.conf` file in livy server and comment out `#spark.master`. 
+
+## Upgrade
+
+* We remove livy.spark.master in zeppelin-0.7. Because we sugguest user to use livy 0.3 in zeppelin-0.7. And livy 0.3 don't allow to specify livy.spark.master, it enfornce yarn-cluster mode.
```
Comment (AhyoungRyu) [inline docs/interpreter/livy.md]: It's just my suggestion; How about putting this sentences under the above Configuartion table with quoted format?

File: livy/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Overview
+Livy interpreter for Apache Zeppelin
+
+# Pre requisities
```
Comment (AhyoungRyu) [inline livy/README.md]: `Prerequisities ` would be better :)

File: docs/interpreter/livy.md (hunk 1 above)
Comment (zjffdu) [inline docs/interpreter/livy.md]: Fixed

**Comment by AhyoungRyu** [comment]
@zjffdu Thanks! Then we can remove `Upgrade` section since we don't need to have both of them :)

File: docs/interpreter/livy.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ Additional requirements for the Livy interpreter are:
 We added some common configurations for spark, and you can set any configuration you want.
 This link contains all spark configurations: http://spark.apache.org/docs/latest/configuration.html#available-properties.
```
Comment (AhyoungRyu) [inline docs/interpreter/livy.md]: If you don't mind, can we change this part to like below? Then ppl don't need to copy & paste manually. 
```
You can find all Spark configurations in [here](http://spark.apache.org/docs/latest/configuration.html#available-properties).
```

Comment (zjffdu) [inline docs/interpreter/livy.md]: Fixed

Comment (AhyoungRyu) [inline docs/interpreter/livy.md]: Thanks!

**Comment by AhyoungRyu** [comment]
Looks good 👍

**Comment by AhyoungRyu** [comment]
Docs change has no test so, CI failure is not related with this change. 
Will merge this if there are no more comments.

**Pull request closed without merging**